app/Console/Commands/SendInventoryAlerts.php

```diff
@@ -45,28 +42,24 @@ public function handle()
     {
         $settings = Setting::getSettings();
 
-        if (($settings->alert_email!='')  && ($settings->alerts_enabled==1)) {
-
+        if (('' != $settings->alert_email) && (1 == $settings->alerts_enabled)) {
             $items = Helper::checkLowInventory();
 
-            // Send a rollup to the admin, if settings dictate
-            $recipient = new \App\Models\Recipients\AlertRecipient();
+            if (($items) && (count($items) > 0) && ('' != $settings->alert_email)) {
+                $this->info(trans_choice('mail.low_inventory_alert', count($items)));
 
-            if (($items) && (count($items) > 0) && ($settings->alert_email!='')) {
+                $recipients = collect(explode(',', $settings->alert_email))->map(function($item, $key) {
+                    return new AlertRecipient($item);
+                });
 
-                $this->info( trans_choice('mail.low_inventory_alert',count($items)) );
-
-                $recipient->notify(new InventoryAlert($items, $settings->alert_threshold));
+                Notification::send($recipients, new InventoryAlert($items, $settings->alert_threshold));
             }
-
         } else {
-            if (Setting::getSettings()->alert_email=='') {
+            if ('' == $settings->alert_email) {
```

Comment:
What's the rationale behind this change? I find it harder to read, but if there's a legitimate reason for reversing this...

Reply:
Hey, sorry about the formatting change. I used a PHP_CS file to standardarize the coding format. I did submit another PR for using a PHP_CS & PHP Mess Detector files. The change is from the coding standard document. I can reverse it it you want. 

And the only comment I know that I removed what * // Send a rollup to the admin, if settings dictate*  which is not true that I can tell from the code. If i'm wrong please let me know and I'll put back.

Reply:
No worries - it just makes merges take longer :)

We typically run PHP_CS and mess detector every now and then in a single PR, so nobody has to visually parse through a whole bunch of formatting changes. I think for the develop branch, I had to pull it at one point because of a dependency conflict (this was a few months ago), but I'm glad to see we can re-add it. 

> And the only comment I know that I removed what * // Send a rollup to the admin, if settings dictate* which is not true that I can tell from the code

That's what that notification does though. The "rollup" means "one single email with all the stuff" versus "one email for each item". Or am I missing something? (It's very possible I'm missing something lol)

Reply:
(We also have mess detection settings in our Codacy settings)

Reply:
> The change is from the coding standard document. I can reverse it it you want.

If you don't mind. I never understood why that was a standard - to me, the variable in question is the "subject" of the sentence, and the value you're evaluating is less important.